projects/fsastorefrontlib/src/cms-components/b2b/user/form/user-form.component.ts

```diff
@@ -46,6 +47,11 @@ export class FSUserFormComponent extends UserFormComponent {
     }
   }
 
+  updateRoles(event: MouseEvent) {
```

Comment:
Overriding Spartacus method that expects `formArray` instead of only one `formControl` radio button. To avoid rewriting the whole logic, we are resetting `formArray` each time radio button is selected so only one can be sent to BE.